his/core/application-integration-importhostdefs.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Importing COBOL Host Definitions"
+description: Importing COBOL Host Definitions for CICS, IMS and Host Files
+author: haroldcampos
+ms.author: "hcampos"
+ms.prod: "host-integration-server"
+ms.topic: how-to
+ms.date: "10/17/2023"
+---
+
+# Importing COBOL Host Definitions
+
+When the original Host Integration Server TI Designer feature was introduced, it was provided with a full set of capabilities to design metadata artifacts from scratch with the assistance of a Mainframe Programmer. The ability to import Host Definitions in the format of a COBOL or RPG copybook was introduced to support scenarios to automate design and to reduce involvement of Mainframe Programmers. Over the years, this became the principal option to create Metadata artifacts.
+
+## Prerequisites
+
+The Main prerequisite is to obtain the Host Definitions (copybooks) that will be imported in the HIS Designer for Logic Apps. The HIS Designer for Logic Apps supports COBOL and RPG copybooks.
+It is also important to understand the Programming model for the technology to integrate against (CICS or IMS). Both platforms have different requirements and ways to pass and receive information. Please get yourself familiar with the appropriate [Programming Model](choosing-the-appropriate-programming-model1.md) before importing the Host Definitions.
```

Comment:
Avoid ["please"](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please), except in specific use cases:

```suggestion
 - Understanding how the programming model works for the technology that you want to integrate against, which is CICS or IMS. Both platforms have different requirements and ways to pass and receive information. Make sure that you learn about the appropriate [programming model](choosing-the-appropriate-programming-model1.md) before you import the host definitions.
```

Reply:
Understood